src/pymor/models/iosys.py

```diff
@@ -928,6 +929,45 @@ def get_ast_spectrum(self, ast_pole_data=None, mu=None):
 
             return ast_lev, ast_ews[idx], ast_rev
 
+    def moebius_substitution(self, M, sampling_time=0):
+        assert isinstance(M, MoebiusTransform)
+        if not isinstance(self.E, IdentityOperator):
+            return NotImplemented
+
+        a, b, c, d = M.coefficients
+        s = a * d - b * c
+        v = np.sqrt(np.abs(s))
+        Gamma = NumpyMatrixOperator(
+            spla.inv(to_matrix(a * self.E - c * self.A)),
```

Comment:
I agree, however, using `InverseOperator` leads to a problem in `LTIModel.transfer_function.tf` where `to_matrix` is called on `D`, because there is no `MatchingRule` for `InverseOperator`...

Reply:
I see. Then it would probably be good for `to_matrix` not to recurse through `ConcatinationOperator`, but instead use something like `concat_op.as_range_array(mu=mu).to_numpy().T` (that way, the `InverseOperator` should not be turned to a matrix).

Reply:
Good idea. I did some small checks and the speeds of both variants are somewhat equal. However, one can encounter problems with large operators, because `as_array_max_length()` defaults to 100. Not a big issue I suspect, because it can be easily raised if needed.